probatus/feature_elimination/feature_elimination.py

```diff
@@ -695,3 +680,156 @@ def plot(self, show=True, **figure_kwargs):
         else:
             plt.close()
         return ax
+
+
+class EarlyStoppingShapRFECV(ShapRFECV):
+    """
+    This is a child of ShapRFECV which allows early stopping during the fitting.
+
+    Early stopping is a feature of models such as as LightGBM or xgboost.
+
+    This class is incompatible with hyperparameter tuning.
+
+    Parent (ShapRFECV) docstring:
+    """
+
+    __doc__ = str(__doc__) + str(ShapRFECV.__doc__)
+
+    def __init__(
+        self,
+        clf,
+        early_stopping_rounds=30,
```

Comment:
What is the reasonable default for this value. What is it set as in LGBm and XGB?
Let's use the ones they use as default.